confluence-xml/src/test/resources/confluencexml/alert.test

```diff
@@ -0,0 +1,91 @@
+.#------------------------------------------------------------------------------
+.expect|filter+xml
+.# Content conversions
+.#------------------------------------------------------------------------------
+<wikiSpace name="MySpace">
+  <wikiDocument name="WebHome">
+    <wikiDocumentLocale>
+      <p>
+        <parameters>
+          <entry>
+            <string>creation_author</string>
+            <string>XWiki.Teo</string>
+          </entry>
+          <entry>
+            <string>creation_date</string>
+            <date>2012-08-21 15:37:47.0 UTC</date>
+          </entry>
+          <entry>
+            <string>lastrevision</string>
+            <string>1</string>
+          </entry>
+        </parameters>
+      </p>
+      <wikiDocumentRevision revision="1">
+        <p>
+          <parameters>
+            <entry>
+              <string>revision_author</string>
+              <string>XWiki.Teo</string>
+            </entry>
+            <entry>
+              <string>revision_date</string>
+              <date>2016-10-11 14:47:37.0 UTC</date>
+            </entry>
+            <entry>
+              <string>revision_comment</string>
+              <string></string>
+            </entry>
+            <entry>
+              <string>title</string>
+              <string>My Space</string>
+            </entry>
+            <entry>
+              <string>content</string>
+              <string>== {{id name="MySpaceHome-alert"/}}alert ==
+
+{{success title="Wohoo! u succeeded" type="Success"}}
```

Comment:
Since we handled the "type" parameter, i dont think we should preserve it anymore.

Reply:
More importantly, it's going to fail because the `type` parameter does not exist in the message macros (or box since that's actually box macro parameters). @Farcasut you might want to test in a real XWiki instance this kind of results when you are not sure what is the behavior.

Reply:
Hi! @tmortagne, the parameter would be ignored, and the macro would be rendered normally, but you are right I should remove it.

Reply:
> the parameter would be ignored

Hmm indeed, seems we removed that check, I had forgotten about that. Anyway, better produce something that follows XWiki specifications.